dts/arm/nxp/nxp_s32z27x_r52.dtsi

```diff
@@ -433,5 +433,32 @@
 				status = "disabled";
 			};
 		};
+
+		i3c0: i3c@401d0000 {
+			compatible = "nxp,s32-i3c";
+			reg = <0x401d0000 0x10000>;
+			#address-cells = <3>;
+			#size-cells = <0>;
+			interrupts = <GIC_SPI 252 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>;
+			status = "disabled";
+		};
+
+		i3c1: i2c@409d0000 {
```

Comment:
typo `i2c@409d0000` should be i3c@409d0000

Reply:
i've updated